### DISCUSSION TITLE
How to remove the left area of a QLayout?

### DISCUSSION THREAD

**Discussion by ZheFrench** [quasarframework/quasar#15201] [General - Components / Directives / etc]
I just want the q-page to fulfill the whole screen (keeping my header and footer). When you don't ask for left or right drawer, you still left unused space on right and left sides.
This is described [here](https://forum.quasar-framework.org/topic/6434/how-to-remove-the-left-area-of-a-qlayout).

**Comment by yusufkandemir** [comment]
No, that's not the case unless you have made some extra configuration, styling, etc. which we can't know because you didn't share any code. It doesn't have any drawers and it takes the full width: https://codepen.io/yusufkandemir/pen/jOpqgZy
